machines/create-connection-url.js

```diff
@@ -0,0 +1,78 @@
+module.exports = {
+//
+//
+  friendlyName: 'Create connection URL',
+//
+//
+  sync: true,
+//
+//
+  description: 'Build a Redis connection URL out of a dictionary of connection options.',
```

Comment:
@sgress454 not to be anal, but at some point, might be worth clarifying this isn't from "a dictionary" per se (e.g. maybe "out of a set of")